src/Files.App/Views/Properties/GeneralPage.xaml

```diff
@@ -379,17 +397,19 @@
 						FontWeight="SemiBold"
 						Text="{helpers:ResourceString Name=PropertiesDriveUsedSpace/Text}" />
 				</StackPanel>
+
 				<TextBlock
 					x:Name="driveUsedSpaceValue"
 					Grid.Row="0"
 					Grid.Column="2"
 					x:Load="{x:Bind ViewModel.DriveUsedSpaceVisibility, Mode=OneWay}"
 					Style="{StaticResource PropertyValueTextBlock}"
 					Text="{x:Bind ViewModel.DriveUsedSpace, Mode=OneWay}" />
+
 				<Button
 					Grid.Row="0"
 					Grid.Column="3"
-					Padding="8,6"
+					Padding="6,6"
```

Comment:
Can you share a screenshot of the before and after?

Reply:
Default button padding is 8,6. but the reason why not 8,8 or 6,6 is to adjust padding for TextBlock (height:width==2:1). however glyph icon is 1:1.

Before/After
![image](https://user-images.githubusercontent.com/62196528/232266025-a172c980-c583-4ac7-9ec7-8740740b2d48.png)
![image](https://user-images.githubusercontent.com/62196528/232266027-252839e8-990a-4397-b851-b334ae296609.png)

Reply:
I think we originally used 8,6 to help center the glyph.

Reply:
This button's flyout can also be TeachingTip.

Reply:
Tooltip is better here because it shows on hover.

Reply:
We can keep this change if you're okay with the glyph being off centered.

Reply:
> Tooltip is better here because it shows on hover.

Tooltip is showed when clicking. so that's the same behaviour as above one.

Reply:
Isn't this flyout?

Reply:
Both are the flyouts. Second one can be replaced with TeachingTip.